src/platform/visualc/config.h

```diff
@@ -68,3 +68,23 @@
 
 #define INLINE __forceinline
 #define DB_FASTCALL __fastcall
+
+/* Definitions of useful mathematical constants under Visual Studio:
+ * M_E        - e
+ * M_LOG2E    - log2(e)
+ * M_LOG10E   - log10(e)
+ * M_LN2      - ln(2)
+ * M_LN10     - ln(10)
+ * M_PI       - pi
+ * M_PI_2     - pi/2
+ * M_PI_4     - pi/4
+ * M_1_PI     - 1/pi
+ * M_2_PI     - 2/pi
+ * M_2_SQRTPI - 2/sqrt(pi)
+ * M_SQRT2    - sqrt(2)
+ * M_SQRT1_2  - 1/sqrt(2)
+ *
+ * Most importantly this provides M_PI, which is typically provided
+ * by GCC and Clang when including <cmath>.
+ */
+#define _USE_MATH_DEFINES
```

Comment:
Instead of listing all constants, perhaps it's better to just link https://docs.microsoft.com/en-us/cpp/c-runtime-library/math-constants in here.

And there's no need of mentioning GCC nor Clang, as this header is MSVC-specific :)

Also, please add a newline after last line (Visual Studio code seems to forget about ending newline char the same way MSVC).

Reply:
Good point, will do!